src/server/services/bodem/loodmetingen.ts

```diff
@@ -174,7 +177,7 @@ export async function getLoodApiHeaders() {
   const requestConfig = getApiConfig('LOOD_365_OAUTH', {
     data,
     headers,
-    cacheKey: `lood-365-oauth-access-token`,
+    cacheKey_UNSAFE: `lood-365-oauth-access-token`,
```

Comment:
Een comment is wel fijn. Dat dit express is.
```suggestion
    cacheKey_UNSAFE: `lood-365-oauth-access-token`, // Following responses will be identical
```